docs/raster/ReadingGeoTiffs.md

```diff
@@ -0,0 +1,88 @@
+## Reading GeoTiffs
+
+- [Introduction](#introduction)
+- [Reading GeoTiffs With GeoTrellis](#reading-geotiffs-with-geotrellis)
+  - [Reading Locally Part 1: Reading For the First Time](#reading-locally-part-1-reading-for-the-first-time)
+  - [Reading Locally Part 2: Expanding Our Vocab](reading-locally-part-2-expanding-our-vocab)
+
+### Introduction
+This tutorial will go over how to read GeoTiff files using GeoTrellis. It assumes that you already have the environment needed to run these examples. If not, please follow this [link] to get GeoTrellis working on your system.
```

Comment:
I humbly suggest a directory hierarchy of:
```
docs
| -- tutorials
| -- guide
| -- architecture
```
so if that's best, `ReadGeoTiffs.md` would live in `docs/tutorials/`, and the `[link]` would be to `setup.md` in the same folder.